### PR TITLE
Add two more examples for arrow use

### PR DESCRIPTION
**Pull request by pdfernhout**
For issue #6

### CODE REVIEW

**Comment by pdfernhout** [comment]
Used wrong email for commit, sorry. Redid the commit with correct email associated with CLA for both Dojo and jQuery foundations and did a force push. Hopefully jquerybot will notice that eventually?

**Comment by pdfernhout** [comment]
I'm sorting out an issue with the name on the CLA with jQuery legal (I embarrassingly fat fingered my last name with two r's), so the CLA process will block this until then

File: STYLE.md [hunk 1]
```diff
@@ -268,6 +268,11 @@ make sense to have the same name, while avoiding shadowing the outer variable.
 		return this.processResult(result);
 	});
 
+	// right
+	promise.then(function (result) => {
```
Comment (kfranqueiro) [inline STYLE.md]: This is invalid syntax :)

**Comment by kborchers** [comment]
@pdfernhout Your incorrect signature has been removed from the database. If you push a new or updated commit (depending on how the project prefers you update a PR) to address the feedback you received, the check should be rerun and it should pass. Let me know if you continue to have issues and thank you for your contribution.

File: STYLE.md (hunk 1 above)
Comment (pdfernhout) [inline STYLE.md]: You're right! Sorry about that leftover from copy and paste from the example above. Thanks for catching that. :-)

**Comment by pdfernhout** [comment]
Fixed the mistake @kfranqueiro pointed out.

**Comment by pdfernhout** [comment]
@kborchers Thanks for fixing my mistake about the CLA so quickly! :-)

**Comment by pdfernhout** [comment]
Moved the new examples from rule 4 to rule 5 where they belong more.

**Pull request closed without merging**